.stylelintrc

```diff
@@ -1,4 +1,5 @@
 {
+  "customSyntax": "postcss-lit",
```

Comment:
Does this exclude any other places where we don't use Lit-like template tagged literals? Can you think of something where we want to lint styles, but use a different syntax?

One thing I was testing is if stylelint reports issues in the following code, but it doesn't (but neither does it on master 🤷‍♂️):
```
  static get template() {
    return html`
      <style>
        ::slotted([slot='value']) { color: green; }
      </style>
```

Reply:
We should generally get rid of those `<style>` tags (and we'll eventually do that when moving to Lit). On master we use [`stylelint-processor-styled-components`](https://github.com/vaadin/stylelint-config-vaadin/blob/fd2a3e368a2798676e83b74a58d10316b3988009/index.js#L4-L16) which works because it happens to support `css` literals. But it stopped working with Stylelint 14 as that major release excluded syntaxes from the core library (you can observe that if you look at the diff in `yarn.lock`).

Reply:
OK, thanks for the clarification. Looks good then.